src/main/java/technology/tabula/extractors/BatchSelectionExtractor.java

```diff
@@ -0,0 +1,430 @@
+package technology.tabula.extractors;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import technology.tabula.ObjectExtractor;
+import technology.tabula.Page;
+import technology.tabula.PageIterator;
+import technology.tabula.Rectangle;
+import technology.tabula.Table;
+import technology.tabula.detectors.NurminenDetectionAlgorithm;
+import technology.tabula.detectors.StringSearch;
+import technology.tabula.detectors.SpreadsheetDetectionAlgorithm;
+import technology.tabula.writers.CSVWriter;
+import technology.tabula.writers.Writer;
+
+// NOTES:
+//		need to remove tables from auto/spread list if they are used as a best guess
+//		or remove very similar tables from the list before extracting data
+public class BatchSelectionExtractor {
```

Comment:
I believe the last section of this is caused by [this section of the code.](https://github.com/dbangera23/tabula-java/blob/master/src/main/java/technology/tabula/extractors/BatchSelectionExtractor.java#L252) I'll switch this to use [Paths.get](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html#get(java.lang.String,%20java.lang.String...)).